usr/share/device-quirks/scripts/aokzoe/aokzoea1ar07.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+### AOKZOE A1 Quirks
+
+if [ $(whoami) != 'root' ]; then
+	echo "You must be root to run this script."
+   exit 1
+fi
+ 
+### Unlock Filesystem
+
+frzr-unlock
+
+### Force 16 bit audio, format S16LE, rate 192000
+
+cp -a /usr/share/wireplumber /etc/
+
+sed -i 's/--\["audio.format"\]/\["audio.format"\]/' /etc/wireplumber/main.lua.d/50-alsa-config.lua
+
+sed -i 's/--\["audio.rate"\]/\["audio.rate"\]/' /etc/wireplumber/main.lua.d/50-alsa-config.lua
+
+sed -i 's/44100/192000/' /etc/wireplumber/main.lua.d/50-alsa-config.lua
+
+ 
+
+### Inject EDID override into initramfs & rotate screen to landscape
```

Comment:
I think we need a compromise here. I'd include this portion into a separate script that can be enabled using a systemd unit by the user manually. @alkazar and @Samsagax will need to buy off on that idea as well. We may also have a more universal way of handling tty rotation we can use.

Reply:
TTY rotation may be workable outside of kernel parameters, but edid injection cannot be done that way as it must be done before drivers are loaded and screen is recognized as connected.

Reply:
I'm not in favour of EDID injection in the kernel... Maybe we can figure it out later.

As for the screen rotation you can write to fbcon rotation file and it will rotate everything: `/sys/class/graphics/fbcon/rotate_all`. You'll need to do that every boot thoug so maybe you can enable/echo/write a tempfile to do it:
https://man.archlinux.org/man/systemd-tmpfiles.8.en

Reply:
> I'm not in favour of EDID injection in the kernel... Maybe we can figure it out later.

I think a reasonable compromise is warranted. We can have a folder, `/usr/share/device-quirks/scripts/user-enabled` that houses things like EDID or DSDT injection that exist and are vetted/tested. The master script can look for entries in `/etc/device-quirks/user-enabled` which the user can create logical links to from the `/usr` dir (to get automatic updates) or write custom scripts. Reenabling the service after linking should handle pushing those changes. I would like to see both the Wiki article covering this and the scripts themselves come with a big warning upfront that this could cause system instability and require a reinstall etc. etc.

@jcfrosty For this PR I'd remove that portion and submit it separate with this framework in mind so the approved changes can be managed while we figure out the remainder in a separate discussion. While frustrating, the more conservative and cautious approach is better for long term stability.